### ISSUE TITLE
Voyager Tables already existing

### ISSUE THREAD

**Issue by obrunsmann** [thedevdojo/voyager#2057]
Hey Guys,

I decided to do not use the issue template, cause my question is more general.

I wanted to integrate voyager in an existing project, the problem ist that I already defined a `categories`-Table as well as a `permissions`-Table cause I use the `spatie/laravel-permission` package.

My firs try was to rename the categories table, but I think I could run into trouble, if I also rename the role tables.

So in short: I do not use all of these generated BREADs like Pages, Categories, etc. ... just wanted to start with a blank admin UI.

**Comment by fletch3555** [comment]
We're working on moving the pages/posts/categories functionality into a Hook.  You should expect that in v1.1 I think. 

As for permissions, those are fairly tightly coupled into Voyager.  Is there a reason you need the spatie package instead of just using Voyager's?

**Comment by obrunsmann** [comment]
Thanks @fletch3555 ! I'm looking forward to v2!! 

Is there anything possible for now? For example just deleting the pages/posts/categories migrations? Especially to avoid the categories conflict? I would think of migrating to the Voyager's role system..

**Comment by emptynick** [comment]
Fixed in https://github.com/the-control-group/voyager/pull/2829
